TLM/TLM/Manager/Impl/VehicleBehaviorManager.cs

```diff
@@ -723,7 +695,7 @@ public class VehicleBehaviorManager : AbstractCustomManager, IVehicleBehaviorMan
 
 		public float ApplyRealisticSpeeds(float speed, ushort vehicleId, ref VehicleState state, VehicleInfo vehicleInfo) {
 			if (Options.realisticSpeeds) {
-				float vehicleRand = 0.01f * (float)GetVehicleRand(vehicleId);
+				float vehicleRand = 0.01f * (float)GetTimedVehicleRand(vehicleId);
```

Comment:
This should not be here. It's part of another bugfix which is not related to the turn-on-red feature